Sources/StreamChat/Models/Attachments/AnyAttachmentPayload.swift

```diff
@@ -6,9 +6,22 @@ import Foundation
 
 /// A protocol an attachment payload type has to conform in order it can be
 /// attached to/exposed on the message.
-public protocol AttachmentPayload: Codable {
+public protocol AttachmentPayload: Codable, Equatable {
     /// A type of resulting attachment.
     static var type: AttachmentType { get }
+
+    /// Update the url of the attachment with the remote one.
+    /// This function will be called once an attachment has successfully uploaded.
+    /// - Parameter url: The remote url of the attachment.
+    mutating func updateRemoteUrl(_ url: URL)
+}
+
+public extension AttachmentPayload {
+    mutating func updateRemoteUrl(_ url: URL) {
```

Comment:
What happens if a customer doesn't implement this method? How does this compare with the current behaviour?

Reply:
Nothing happens